src/bin/importSqlFile.js

```diff
@@ -71,14 +72,12 @@ const unescape = (val) => {
   await util.promisify(db.init.bind(db))();
   log('done');
 
-  log('open output file...');
-  const lines = fs.readFileSync(sqlFile, 'utf8').split('\n');
+  log(`Opening ${sqlFile}...`);
+  const stream = fs.createReadStream(sqlFile, {encoding: 'utf8'});
 
-  const count = lines.length;
+  log(`Reading ${sqlFile}...`);
   let keyNo = 0;
-
-  process.stdout.write(`Start importing ${count} keys...\n`);
-  lines.forEach((l) => {
+  for await (const l of readline.createInterface({input: stream, crlfDelay: Infinity})) {
     if (l.substr(0, 27) === 'REPLACE INTO store VALUES (') {
       const pos = l.indexOf("', '");
```

Comment:
Here I find the following pattern to be present as a deliminator in the SQLite dump:

```js
const pos = l.indexOf("','");
```